changelog/changelog.ddoc

```diff
@@ -8,8 +8,7 @@ CHANGELOG_NAV_FIRST=$(DIVC changelog-nav, next version: $(A $1.html, $1))
 CHANGELOG_NAV_LAST=$(DIVC changelog-nav, previous version: $(A $1.html, $1))
 
 SUBNAV=$(SUBNAV_TEMPLATE
-    $(SUBNAV_HEAD Change Log, $(ROOT_DIR)changelog/index.html, List of all versions,
-        subnav-duplicate)
+    $(SUBNAV_HEAD Change Log, $(ROOT_DIR)changelog/index.html, List of all versions)
```

Comment:
Alternatively, I think we can change the definition of `SUBNAV_HEAD` from:
```
SUBNAV_HEAD=
    $(DIVC head,
        $(H2 $1)
        $(TC p, $4,
            $(LINK2 $2, $3))
```
to:
```
SUBNAV_HEAD=
    $(DIVC head,
        $(H2 $1)
        $(P
            $(LINK2 $2, $3))
```

though I haven't tested this yet.